docs/installation-guide-mac.rst

```diff
@@ -0,0 +1,83 @@
+:orphan:
+
+Installation guide for macOS
+============================
+
+Follow these instructions to install the CityEnergyAnalyst (CEA) on a macOS system (tested with Mojave 10.14.5) from the source
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Download and install `Git`_ (64-bit).
+* Download and install `GitHub Desktop`_ (64-bit).
+
+  * (this is not strictly necessary, but we find it easier for novice git users)
+
+* Download and install `Miniconda`_ (64-bit) for Python 2.7.
+
+  * (the Mac OS X 64-bit `.pkg installer`_ is probably the easiest way to install Miniconda on your Mac)
+
+* Download and install `CMake`_.
+
+  * (CMake is required for compiling DAYSIM)
+  * (this guide assumes you installed CMake to ``/Applications/CMake`` and have it in your PATH)

```

Comment:
What do you mean by this? Is CMake a directory or an application? By default, CMake gets installed to ``/Applications/CMake.app``. Also, how is CMake added to the ``PATH``?

Reply:
Also, how is CMake added to the ``PATH``?